packages/ifpack2/test/belos/CMakeLists.txt

```diff
@@ -358,6 +358,7 @@ TRIBITS_ADD_TEST(
   COMM serial mpi
   NUM_MPI_PROCS 2
   STANDARD_PASS_OUTPUT
+  EXCLUDE_IF_NOT_TRUE ${PACKAGE_NAME}_ENABLE_Zoltan2Core ${PACKAGE_NAME}_ENABLE_Xpetra
```

Comment:
Doesn't matter to me either way.  I had assumed the TriBITS way would notify me that it was disabling the test and why, but since it doesn't seem to do that, this is fine by me.  I did indent the body, it seems that some places in this file do indent and some don't.